.nycrc

```diff
@@ -0,0 +1,14 @@
+{
+    "check-coverage": true,
+    "lines": 68.16,
+    "statements": 66.1,
+    "functions": 81.82,
+    "branches": 66.28,
+    "reporter": [
+      "lcov",
+      "text"
```

Comment:
the `lcov` reporter seems to generate html as well `coverage/lcov-report/index.html`. Is that enough?
<img width="1189" alt="screen shot 2017-04-20 at 10 16 16" src="https://cloud.githubusercontent.com/assets/3098521/25220585/8196eb04-25b2-11e7-8fd8-419bfe9cff2d.png">